core/platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/shared/Constants.java

```diff
@@ -36,8 +36,10 @@
     public static final String LINK_REL_GET_MACHINE_LOGS_CHANNEL   = "get machine logs channel";
     public static final String LINK_REL_GET_MACHINE_STATUS_CHANNEL = "get machine status channel";
 
-    public static final String WS_AGENT_SERVER_REFERENCE = "extensions";
+    public static final String WS_AGENT_REFERENCE                  = "workspace-agent";
+    public static final String WS_AGENT_DEBUG_REFERENCE            = "workspace-agent-debug";
```

Comment:
@garagatyi does this doc https://wiki.codenvycorp.com/display/PSR/REST+services+links impact reference's name (using dots as word separator) as well?